Podfile.lock

```diff
@@ -200,4 +200,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 484a1cdf04951cc82156f29de196efd76d3ad893
 
-COCOAPODS: 1.15.2
+COCOAPODS: 1.14.3
```

Comment:
Nah, it was just out of sync with the [Gemfile](https://github.com/element-hq/element-ios/blob/bb8b3d9c0d61e6625f160bdf6681d5cec12a8697/Gemfile#L5). That happens when someone runs `pod install` instead of `bundle exec pod install`.